src/cmd/manage_nsfs.js

```diff
@@ -302,6 +303,31 @@ async function add_bucket(data) {
     write_stdout_response(ManageCLIResponse.BucketCreated, data_json);
 }
 
+/** verify_bucket_owner will check if the bucket_owner has an account
+ * in case it does not find one, it would throw an error
+ * @param {string} bucket_owner
+ */
+async function verify_bucket_owner(bucket_owner) {
+    let is_bucket_owner_exist = false;
+    const show_secrets = false;
+    const fs_context = native_fs_utils.get_process_fs_context();
+    const entries = await nb_native().fs.readdir(fs_context, accounts_dir_path);
+    await P.map_with_concurrency(10, entries, async entry => {
+        if (entry.name.endsWith('.json') && !is_bucket_owner_exist) {
```

Comment:
why not just throw? even if found bucket_owner === data.email you are still going to iterate the array and check this condition

Reply:
I will iterate the array, that's true, but I will not execute the "expensive" operation of `get_config_data`.
If I find that the bucket_owner exists I don't throw the error, I just stop opening the files and read them.

Reply:
there is no reason to keep iterating...

Reply:
You're right. But map is not iterating under the hood (and hence I cannot stop/break it), it transforms every element and I'm using it for the concurrency that is already implemented.
This implementation would change after the account identifier would be ID that would pass and the account will be lookup by this ID.
In any way, I would add this gap in the PR and inside the code.

Reply:
in verify_delete_account(), you threw an error in the middle of the loop, why not throw an error here as well?

Reply:
Decided that I would add this as GAP, and we will fix it after the dev freeze (improve performance).